### ISSUE TITLE
Is there a way to keep array

### ISSUE THREAD

**Issue by winjeysong** [sindresorhus/dot-prop#117]
Thanks for maintaining this library, and It helps me a lot. But is there a way to keep array like below:

```js
import { deepKeys, getProperty } from 'dot-prop';

const o = { a: ['foo', 'bar'], b: { foo: 'bar' }, c: { d: ['foo', 'bar'] } }
deepKeys(o).forEach(key => {
  console.log(key, ': ', getProperty(o, key))
})

// logs: 
//=> a: ['foo', 'bar']
//=> b.foo: bar
//=> c.d: ['foo', 'bar']
```

**Comment by sindresorhus** [comment]
The point of `deepKeys` is to get the path to all properties. Just curious, what's your use-case? I guess it would be possible to add an option to stop at arrays.

Would you expect `{a: [{c: true}, {d: false}]}` to not include the paths for `c` and `d`?

**Comment by winjeysong** [comment]
Yes, it might be a good idea to add an option to keep arrays, so that I can replace with new array directly, but not set array items one by one.


> The point of `deepKeys` is to get the path to all properties. Just curious, what's your use-case? I guess it would be possible to add an option to stop at arrays.
> 
> Would you expect `{a: [{c: true}, {d: false}]}` to not include the paths for `c` and `d`?
